sssd_test_framework/utils/authentication.py

```diff
@@ -688,25 +738,46 @@ def password(self, username: str, password: str) -> bool:
         rc, _, _, _ = self.password_with_output(username, password)
         return rc == 0
 
-    def password_expired(self, username: str, password: str, new_password: str) -> bool:
+    def password_expired_with_output(
```

Comment:
This function is duplicated in the previous class. Can it be set in an intermediate class which both classes inherit from?

Reply:
Let me think about this some more. Can we work with this for now, and I'll make change based upon this being in master?

I need to update the expect 'passwd' change functions as well, it can certainly be improved.